### PR TITLE
wrappers/cross-cc: drop -isystem /usr/include, too

### PR DESCRIPTION
**Pull request by sgn**
For #46246 
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: common/wrappers/cross-cc [hunk 1]
```diff
@@ -23,13 +23,13 @@ while [ $i -lt ${#ARGS[@]} ]; do
 			MYARGS+=("-L${arg}")
 		fi
 		unset libpath
-	elif [ "$arg" = "-I" ]; then
+	elif [ "$arg" = "-I" -o "$arg" = "-isystem" ]; then
```
Comment (Johnnynator) [inline common/wrappers/cross-cc:26]: This changes all `-isystem` to `-I`, which changes the search order that might be used for headers.

**Comment by SpidFightFR** [comment]
+up (to enable notifications 😅)

**Comment by classabbyamp** [comment]
> +up (to enable notifications 😅)

there's a subscibe button in the sidebar

**Comment by SpidFightFR** [comment]
Oh... :sweat_smile: 
thanks

**Pull request merged**